pkg/util/http/transport.go

```diff
@@ -55,6 +55,33 @@ func warnOnce(warnMap map[string]bool, key string, format string, params ...inte
 	}
 }
 
+// minTLSVersionFromConfig determines the minimum TLS version defined by the given
+// config, accounting for defaults and deprecated configuration parameters.
+//
+// The returned result is one of the `tls.VersionTLSxxx` constants.
+func minTLSVersionFromConfig(cfg config.Config) uint16 {
+	var min uint16
+	switch cfg.GetString("min_tls_version") {
+	case "1.0":
+		min = tls.VersionTLS10
+	case "1.1":
+		min = tls.VersionTLS11
+	case "1.2":
+		min = tls.VersionTLS12
+	case "1.3":
+		min = tls.VersionTLS13
+	case "": // default
+		if cfg.GetBool("force_tls_12") {
+			min = tls.VersionTLS12
+		} else {
+			min = tls.VersionTLS10
+		}
+	default:
+		log.Warnf("Invalid `min_tls_version` %#v", cfg.GetString("min_tls_version"))
```

Comment:
Which version will be used in this case? Would it make sense to make it part of the log message?